front/app/components/UI/Image/index.tsx

```diff
@@ -115,21 +95,8 @@ export default class Image extends PureComponent<Props, State> {
         onLoad={this.handleImageLoaded}
         id={id}
         className={className || ''}
+        loading={isLazy ? 'lazy' : 'eager'}
       />
     );
-
-    if (cover) {
-      image = <Fallback src={src} className={className} />;
-    }
-
-    if (isLazy) {
-      return (
-        <Observer rootMargin="200px" onChange={this.handleIntersection}>
-          {image}
-        </Observer>
-      );
```

Comment:
I don't understand what the use of this was. We set visible to true, which then assign  src to ImageElement's src prop. I couldn't find online how this would lazyload the image, so I instead am using the built-in `loading` attribute now which has [reasonable support](https://caniuse.com/?search=lazyloading).

We could go [further](https://www.freecodecamp.org/news/how-to-lazy-load-images-in-react/#:~:text=To%20start%20using%20it%2C%20instead,attribute%20as%20you%20would%20normally.&text=Also%2C%20we%20explicitly%20defined%20the,cumulative%20layout%20shift(CLS).), but given I think it was not working before and I don't want to risk going down rabbit holes, I'm leaving it at that.